tests/integration-tests/tests/update/test_update/test_dynamic_file_systems_update/pcluster.config.update_drain.yaml

```diff
@@ -0,0 +1,103 @@
+Image:
+  Os: {{ os }}
+HeadNode:
+  InstanceType: {{ instance }}
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+  Iam:
+    S3Access:
+      - BucketName: {{ bucket_name }}
```

Comment:
Why does the head node require read access to the bucket if we do not execute any explicit call to AWS S3 targeting that bucket? Same question for compute nodes below.

Reply:
It is used in the test: `sudo aws s3 cp --region`. Test will fail without specifying the s3 bucket